test/test_checks.py

```diff
@@ -9,6 +24,30 @@ def test_get_noqa_lines(self):
         self.assertEqual(checker.get_noqa_lines(checker.get_file_contents()), [2])
 
 
+class TestFlake8Stdin(TestCase):
+
+    @expectedFailureIf(not FLAKE8_2)
+    def test_stdin(self):
+        """Test using stdin."""
+        filename = get_absolute_path('data/doubles.py')
+        env = os.environ.copy()
+        env['PYTHONIOENCODING'] = 'utf8'
+        with open(filename, 'rb') as f:
+            # For some reason using "--select=Q" did suppress all outputs, so
+            # the result might contain non flake_quotes related errors
+            p = subprocess.Popen(['flake8', '-'], stdin=f, env=env,
+                                 stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            out, err = p.communicate()
+
+        out = out.decode('utf8').splitlines()
+        self.assertFalse(err)
```

Comment:
Yes `assertFalse` does not check `VALUE is False` not `not VALUE`, which an empty string is as well. (to actually check if the value is False `assertIs(VALUE, False)`)